index.html

```diff
@@ -105,6 +107,29 @@ <h3>
           reaches its maximum capacity and gets flushed.
         </p>
       </dd>
+      <dt>
+        <dfn><code>jitterBufferReceivedPackets</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long-long">unsigned long long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Experimental stat which will be available under origin trial in
```

Comment:
Is there a link to the origin trial discussion or something saying which origin trial?

Reply:
https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/hE2B1iItPDk/discussion